crypto-ffi/src/generic.rs

```diff
@@ -704,7 +704,7 @@ impl core_crypto::prelude::CoreCryptoCallbacks for CoreCryptoCallbacksWrapper {
 }
 
 /// This only exists to create a sync interface to our internal async callback interface
-// TODO: Remove this once UniFFI supports async callbacks
+// TODO: Remove this once UniFFI supports async callbacks. Tracking issue: WPB-9588
```

Comment:
I suggest rewording the comment since apparently UniFFI supports this now (also see the tracking issue).